includes/class-wp-job-manager-forms.php

```diff
@@ -50,7 +50,7 @@ public function __construct() {
 	 */
 	public function load_posted_form() {
```

Comment:
I was thinking to rename this method, but I think it's not too weird to justify a deprecation only to rename the method. WDYT?

Reply:
Let's leave it like this.